### PR TITLE
Update UmbracoNetCoreUpdates.md

### PR DESCRIPTION
**Pull request by bergmania**
Added info about the nightly builds

### CODE REVIEW

File: UmbracoNetCoreUpdates.md [hunk 1]
```diff
@@ -146,3 +146,23 @@ The project is now running on the [Kestrel server](https://docs.microsoft.com/en
 The next step is to run through the Umbraco CMS installation. If you chose to use MS SQL Server/Azure you will need to add your connection string during this setup process.
 
 Once the installation process is complete you might need to **manually restart the application** in order to start the application again and get access to the Umbraco backoffice.
+
+## .NET Core Nightly Builds
+To get the nightly builds, you will need to add another nuget source
```
Comment (sofietoft) [inline UmbracoNetCoreUpdates.md:151]:
```suggestion

To get the latest nightly builds - the latest version of the Umbraco dotnet template, you will need to add another NuGet source. 
```

File: UmbracoNetCoreUpdates.md [hunk 2]
```diff
@@ -146,3 +146,23 @@ The project is now running on the [Kestrel server](https://docs.microsoft.com/en
...
+
+1. Install the new Umbraco dotnet template:
+
+You will have to get the latest template from the new source. To do this, you will need to use a wildcard symbol
+
+
+    ```none
+    dotnet new -i Umbraco.Templates::0.5.0-*
+    ```
+
```
Comment (sofietoft) [inline UmbracoNetCoreUpdates.md:167]:
```suggestion
2. Install the new Umbraco dotnet template
    ```none
    dotnet new -i Umbraco.Templates::0.5.0-*
    ```

In order to get the latest template from the new source, you will need to use a wildcard symbol like shown above.

```

File: UmbracoNetCoreUpdates.md [hunk 3]
```diff
@@ -146,3 +146,23 @@ The project is now running on the [Kestrel server](https://docs.microsoft.com/en
...
+1. Install the new Umbraco dotnet template:
+
+You will have to get the latest template from the new source. To do this, you will need to use a wildcard symbol
+
+
+    ```none
+    dotnet new -i Umbraco.Templates::0.5.0-*
+    ```
+    
+Now you can continue from as using the [Alpha version](#steps-to-create-an-umbraco-solution-using-the-dotnet-new-template)
```
Comment (sofietoft) [inline UmbracoNetCoreUpdates.md:168]:
```suggestion
Now you can continue using the [Alpha version](#steps-to-create-an-umbraco-solution-using-the-dotnet-new-template).
```

**Review by sofietoft — COMMENTED** [review]
Thanks for adding this @bergmania .

I've made a few suggestions.

File: UmbracoNetCoreUpdates.md (hunk 2 above)
Comment (bergmania) [reply UmbracoNetCoreUpdates.md:168]: Bør det ikke stadig være `1.` ? Det bør være noget markdown håndtere at ændre til det korrekte nummer

File: UmbracoNetCoreUpdates.md (hunk 3 above)
Comment (bergmania) [reply UmbracoNetCoreUpdates.md:169]: Hmm.. Det er ikke helt det jeg mener.. "Det er på samme måde som ved alpha" Man bruger ikke alpha

Comment (sofietoft) [reply UmbracoNetCoreUpdates.md:169]: Alright. Then I am not entirely sure I understand the sentence.
"Now you can continue from as using the Alpha version"
What exactly does that mean then? 🤔

File: UmbracoNetCoreUpdates.md (hunk 2 above)
Comment (sofietoft) [reply UmbracoNetCoreUpdates.md:168]: Yes, I thought so as well.
But it looks like there are too many elements between two to list items, and it then acts like two separate lists.
Which is why I updated it.

File: UmbracoNetCoreUpdates.md (hunk 3 above)
Comment (bergmania) [reply UmbracoNetCoreUpdates.md:169]:
```suggestion
Now you can continue in the same way as if you were using the [Alpha version](#steps-to-create-an-umbraco-solution-using-the-dotnet-new-template)
```

Better?

Comment (sofietoft) [reply UmbracoNetCoreUpdates.md:167]: Ah, yes. Completely misunderstood that! Looks good, thanks!

**Pull request merged**